.circleci/cb-publish-step-2-verdaccio.sh

```diff
@@ -8,7 +8,7 @@ function lernaPublishExitOnFailure {
   set -e
   # run lerna publish with the args that were passed to this function
   # duplicate stdout to a temp file
-  npx lerna publish "$@" | tee /tmp/publish-results
+  npx lerna publish "$@" 2>&1 | tee /tmp/publish-results
```

Comment:
Apparently error message got lost here.
See https://serverfault.com/questions/201061/capturing-stderr-and-stdout-to-file-using-tee .

From failed attempt:
```
lerna notice shasum:        fa056e4b14aeec3aef7fa720618f41805b4a9833     
lerna notice integrity:     sha512-fLR1TEyNkFv2y[...]t46eMiwPA2TEQ==     
lerna notice total files:   102                                          
lerna notice 
lerna ERR! ECONNRESET request to https://registry.npmjs.org/amplify-nodejs-function-runtime-provider failed, reason: read ECONNRESET
lerna ERR! FetchError: request to https://registry.npmjs.org/amplify-nodejs-function-runtime-provider failed, reason: read ECONNRESET
lerna ERR!     at ClientRequest.<anonymous> (/codebuild/output/src2344/src/github.com/aws-amplify/amplify-cli/node_modules/npm-registry-fetch/node_modules/minipass-fetch/lib/index.js:130:14)
lerna ERR!     at ClientRequest.emit (node:events:513:28)
lerna ERR!     at TLSSocket.socketErrorListener (node:_http_client:502:9)
lerna ERR!     at TLSSocket.emit (node:events:525:35)
lerna ERR!     at emitErrorNT (node:internal/streams/destroy:151:8)
lerna ERR!     at emitErrorCloseNT (node:internal/streams/destroy:116:3)
lerna ERR! lerna request to https://registry.npmjs.org/amplify-nodejs-function-runtime-provider failed, reason: read ECONNRESET
Success! Lerna publish succeeded.
```